runtime/trappist/Cargo.toml

```
@@ -96,6 +96,7 @@ xcm-primitives = { path = "../../primitives/xcm", default-features = false }
 # External Pallets
 pallet-dex = { version = "0.0.1", git = "https://github.com/paritytech/substrate-dex.git", default-features = false }
 pallet-dex-rpc-runtime-api = { version = "0.0.1", git = "https://github.com/paritytech/substrate-dex.git", default-features = false }
+pallet-chess = { git = "https://github.com/SubstrateChess/pallet-chess.git", default-features = false, branch = "master" }
```

```suggestion
pallet-chess = { git = "https://github.com/SubstrateChess/pallet-chess.git", default-features = false, branch = "polkadot-v0.9.37" }
```

I have agreed with @bernardoaraujor to create releases branches so any project wanting to use this pallet can rely on different polkadot/substrate releases.

---

i would do the same for the dex pallet as well.